src/features/product-card-to-cart/ui/ProductCardToCart/ProductCardToCart.tsx

```diff
@@ -40,6 +48,8 @@ const ProductCardToCart = () => {
     setIsLiked(!isLiked);
     setLoading(true);
 
+    ShowMessage('Product added to Cart ', options);
+
```

Comment:
ShowMessage should transmitted via props to avoid coupling with a component becuz this component may called without popup